packages/frontend/src/styles.scss

```diff
@@ -9,3 +9,16 @@ body {
   margin: 0;
   font-family: Roboto, "Helvetica Neue", sans-serif;
 }
+
+a {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+form .mat-form-field-underline {
```

Comment:
Let's put it where form component is

Reply:
I think it's won't work  for mat-form-field-underline from component styles with component default encapsulation. However I'll check to make sure. Or maybe there are other solutions?